frontend/src/views/PrinterControlPage.vue

```diff
@@ -112,25 +112,32 @@
           </div>
         </div>
         <div class="stream-container">
-          <div v-if="webcams.length > 1" class="webcam-selector">
-            <select id="webcam-select" v-model="selectedWebcamIndex" class="custom-select">
-              <option v-for="(webcam, index) in webcams" :key="index" :value="index">
-                {{ webcam.name || '_default_' }}
-              </option>
-            </select>
+          <div class="header-container">
+            <div class="title">{{ $t('Webcam') }}</div>
+            <div class="webcam-options">
```

Comment:
Where is webcam-options defined? If it's not defined, it means it's not used. Just remove it in this case.